### DISCUSSION TITLE
23.11.20 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#71] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 우재님 코드 리뷰
    - 테스트 코드 작성
- 내일은 어떤 일을 하나요?
    - 테스트 마저 코드 작성

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 리뷰 내용 수정, PR 확인
  - 도메인 병합(진행중)
    - 멤버에서 아티스트 좋아요 합치는 중입니다
- 내일은 어떤 일을 하나요?
  - PR merge
  - 도메인 병합
